tests/rest/admin/test_statistics.py

```diff
@@ -92,7 +92,7 @@ def test_invalid_parameter(self) -> None:
             channel.code,
             msg=channel.json_body,
         )
-        self.assertEqual(Codes.INVALID_PARAM, channel.json_body["errcode"])
+        self.assertEqual(Codes.UNKNOWN, channel.json_body["errcode"])
```

Comment:
It's unfortunate that the errcode is now `UNKNOWN`. Is it something that can be fixed easily?

Reply:
Before, the error itself was generated in `statistics.py` and now the already existing function  `parse_string` is used. There it is no errcode implemented.

https://github.com/matrix-org/synapse/blob/e8ae94a22367a81049582dfdb16c743a45ca4e8b/synapse/http/servlet.py#L383-L388

Reply:
I see!

Would it make sense to set the errcode for the SynapseError?
```py
raise SynapseError(400, message, Codes.INVALID_PARAM)
```

Reply:
IMO, yes. But this should be a new issue / PR. Probably there are a lot of changes to the unit tests.